### ISSUE TITLE
File `beamer.cls' not found. 

### ISSUE THREAD

**Issue by clukewatson** [texstudio-org/texstudio#1268]
### Environment
- TeXstudio: 3.0.1 (git 3.0.1)
- Qt: Using Qt Version 5.15.0, compiled with Qt 5.15.0 R
- OS: macOS Catalina
- TeX distribution: TeX Live 2018 / 2020 (upgraded in trying to solve, but did not work)

### Background

I do not know much about computers; I just like to use LaTeX for writing technical documents.

I never had an issue until this week.
I am teaching a class and I wanted them to use Rmarkdown for assignments.
I had them install ``tinytex'' package in R, which I did along with them.
I did not think about it at the time, but this may have been a bad idea and now Texstudio cannot compile beamer.

I removed old TeX Live 2018 and upgraded to 2020; I also uninstalled Texstudio and reinstalled but the issue remains.

### Expected behavior
Open Texstudio.
File -> New from Template -> Beamer.
Compile.
Produce a beamer file.

### Actual behavior

I get ``File `beamer.cls' not found. \usepackage'' as error log.
No preview or PDF is produced.

**Comment by foreachthing** [comment]
This works in Windows 10 x64, MikTex and TeXstudio 3.0.1:
```
\documentclass{beamer} % https://ctan.org/pkg/beamer
\usetheme{default}

\title{Beamer Template}
\author{TeXstudio Team}
\begin{document}
\begin{frame}[plain]
    \maketitle
\end{frame}
\begin{frame}{Frame Title}
\end{frame}
\end{document}
```

You could try your document here: https://www.overleaf.com/ (zip your project and upload it)

**Comment by sunderme** [comment]
sounds like your tex installation is missing the beamer package.
Usually you can select and install that package.
This is not a txs issue...

**Comment by clukewatson** [comment]
Sorry for not responding sooner. I am teaching and had an external presentation this week so did not have time to solve the issue.

1. I know that there is nothing wrong with the documents because I uploaded everything to overleaf so I got get my work done 
2. I've removed MacTeX and redownloaded it; **isn't Beamer a part of that installation**?
3. I removed TexStudio and redownloaded it; shouldn't that have made it look 'in the right places' ?
